frontend/src/components/UserlocationCard.tsx

```diff
@@ -0,0 +1,58 @@
+import {Userlocation} from "./Entities.ts";
+
+type UserlocationCardProps = {
+    userlocation: Userlocation
+}
+
+export default function UserlocationCard (props: UserlocationCardProps) {
+
+    return (
+        <div id="edit-container">
+
+            These are the credentials of your individual location.
+            Based on the information you're providing below, we are aiming to recommend the perfect groupset for your road bike.
+
+            <form>
+                <p>{props.userlocation.id}</p>
+                <div className="label-input">
```

Comment:
```suggestion
                <div className="label-input margin-botton">
```
Du kannst mehrere Klassen mit Leerzeichen getrennt hier schreiben und alle werden auf diesen div angewendet.
Darüber kannst du gezielt Abstände z.B. nach unten regeln.